docs/integrations/creating-nodes/plan/node-types.md

```diff
@@ -4,14 +4,24 @@ contentType: explanation
 
 # Node types: Trigger and Action
 
-There are two node types you can build for n8n: app nodes and trigger nodes.
+There are two node types you can build for n8n: trigger nodes and action nodes.
```

Comment:
What about AI nodes?

Reply:
Good point, what do you think of adding AI nodes on this page @StarfallProjects ?

Reply:
Are AI nodes basically action nodes though? Or call them cluster nodes?

Reply:
I think AI nodes are technically part of action nodes, and we also don't have a tutorial for AI related nodes (or a starter template) right now, so will leave this out for the time being.